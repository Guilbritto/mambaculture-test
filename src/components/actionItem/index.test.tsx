import React from 'react'
import { render } from '@testing-library/react'
import {ActionItem} from './'
import jest from 'next/jest'


describe('ActionItem Component', () => {
    it('should match the snapshot', () => {
        const { asFragment } = render(<ActionItem label='' onClick={() => jest()} />)
        expect(asFragment()).toMatchSnapshot()
    })
})