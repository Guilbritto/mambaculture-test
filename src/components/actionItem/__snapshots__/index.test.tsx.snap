// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionItem Component should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="border-b flex items-center gap-2 p-3 cursor-pointer hover:bg-slate-100 font-normal space-x-2"
  />
</DocumentFragment>
`;
